--- a/configure.ac	2018-06-11 16:50:05.149311765 -0400
+++ b/configure.ac	2018-06-11 16:50:38.202898725 -0400
@@ -1032,12 +1032,12 @@
 
 	xft_found=no
 	if test x$enable_xft != xno; then
-	    AC_PATH_PROG(FTCONFIG, freetype-config)
+	    AC_PATH_PROG(FTCONFIG, pkg-config)
 
 	    if test "x$FTCONFIG" != x; then
-		CPPFLAGS="`$FTCONFIG --cflags` $CPPFLAGS"
-		CFLAGS="`$FTCONFIG --cflags` $CFLAGS"
-		CXXFLAGS="`$FTCONFIG --cflags` $CXXFLAGS"
+		CPPFLAGS="`$FTCONFIG --cflags xft` $CPPFLAGS"
+		CFLAGS="`$FTCONFIG --cflags xft` $CFLAGS"
+		CXXFLAGS="`$FTCONFIG --cflags xft` $CXXFLAGS"
 
 		AC_CHECK_LIB(fontconfig, FcPatternCreate)
 		AC_CHECK_HEADER(X11/Xft/Xft.h,

