--- CMakeLists.txt	2018-04-24 04:43:11.000000000 -0400
+++ CMakeLists.txt	2018-06-07 19:27:22.637192134 -0400
@@ -88,12 +88,12 @@
 
 
 add_executable(Cutter ${UI_FILES} ${QRC_FILES} ${SOURCE_FILES} ${HEADER_FILES})
-qt5_use_modules(Cutter Core Widgets Gui Svg)
+target_link_libraries(Cutter Qt5::Core Qt5::Widgets Qt5::Gui Qt5::Svg)
 target_link_libraries(Cutter ${RADARE2_LIBRARIES})
 if(CUTTER_ENABLE_JUPYTER)
     target_link_libraries(Cutter ${PYTHON_LIBRARIES})
     if(CUTTER_ENABLE_QTWEBENGINE)
+        target_link_libraries(Cutter Qt5::WebEngineWidgets)
-        qt5_use_modules(Cutter WebEngineWidgets)
     endif()
 endif()
