--- a/Doxyfile.in	2017-06-02 11:54:06.000000000 -0400
+++ b/Doxyfile.in	2017-10-02 14:51:39.302324598 -0400
@@ -83,7 +83,8 @@
 # will be relative from the directory where doxygen is started.
 # This tag requires that the tag FULL_PATH_NAMES is set to YES.
 
-STRIP_FROM_PATH        =
+STRIP_FROM_PATH        = @PROJECT_SOURCE_DIR@ \
+                         @PROJECT_BINARY_DIR@
 
 # The STRIP_FROM_INC_PATH tag can be used to strip a user-defined part of the
 # path mentioned in the documentation of a class, which tells the reader which

